test/ex_doc/formatter/html/erlang_test.exs

```diff
@@ -15,13 +15,13 @@ defmodule ExDoc.Formatter.HTML.ErlangTest do
 
     %% @doc
     %% f/0 function.
-    -spec foo(atom()) -> atom().
+    -spec foo(t:atom()) -> t:atom().
```

Comment:
@garazdawi this change prints warnings:

```
warning: documentation references type "t:atom/0" but it is undefined or private
```

and they were not caught manually by myself. 😅 Just making sure, we need to use `t:` in _markdown_ docs but we don't need to do so in -type and -spec expressions. Is it OK if I revert this change? Or you meant to test atom() type on a t module?

Reply:
ah, I see. the purpose of the test was to ensure we link to erlang.org/doc for these. I'll investigate other breakages. Thanks!

Reply:
oh, we did _not_ link `atom()` and friends to erlang.org but now we _do_ and that's why tests broke. I'll update accordingly!